docs/articles/nunit/Towards-NUnit4.md

```diff
@@ -208,7 +212,7 @@ Add or modify your `nuget.config` to add the package resources such as below:
     <packageSources>
         <clear/>
         <add key="nuget" value="https://api.nuget.org/v3/index.json" />
-        <add key="Myget" value="https://nuget.pkg.github.com/nunit/index.json" />
+        <add key="Github" value="https://nuget.pkg.github.com/nunit/index.json" />
```

Comment:
❓ I receive a password prompt when visiting this GitHub packages URL. Do users need a valid GitHub login in order to access this feed? If so, we should probably mention that.

Reply:
Thanks @SeanKilleen for mentioning this. When `MyGet` was done we tried [`Github`](https://github.com/nunit/nunit/pull/4443). It worked for me, but if it is not publicly available for non-github accounts I see not much point in pursuing this further.

Reply:
Apparently we can indeed create public packages. Just need to allow it and then confirm we're doing so. 👍